src/firefly/java/edu/caltech/ipac/firefly/server/query/lsst/LSSTCatalogSearch.java

```diff
@@ -331,7 +331,7 @@ public void prepareTableMeta(TableMeta meta, List<DataType> columns, ServerReque
             }
 
             // only set for image meta table
-            meta.setAttribute(MetaConst.DATASET_CONVERTER,
+            meta.setAttribute(MetaConst.IMAGE_SOURCE_ID,
                     (String)LSSTQuery.getDatasetInfo(catTable, new String[]{MetaConst.DATASET_CONVERTER}));
```

Comment:
MetaConst.DATASET_CONVERTER -> MetaConst.IMAGE_SOURCE_ID ?

Reply:
Yes, we are deprecating `DATASET_CONVERTER` and using `IMAGE_SOURCE_ID`

Reply:
fixed.